packages/main/src/resolvers/conversation.ts

```diff
@@ -66,7 +66,8 @@ export const Conversation: ConversationResolvers = {
     const presentables = C.getPresentableElements(conversation)
     const latest = presentables.last(null)
     const content = latest && Seq(latest.contents).first(null)
-    const visible = content && replyParser(content.content, true)
+    const visible =
+      content && replyParser(content.content ? content.content : "", true)
```

Comment:
I could have used a more descriptive variable name to make the intent clear. `replyParser` removes stuff like quoted replies from message content. If there is no content, then there is no point in invoking `replyParser` with an empty string.

In this case if `content.content` is `null` or `undefined` then I would like this entire function to return `null` (or `undefined`).